xla/stream_executor/gpu/gpu_command_buffer_test.cc

```diff
@@ -673,13 +693,13 @@ TEST(CudaCommandBufferTest, ConditionalWhile) {
 
   {  // Load addition kernel.
     MultiKernelLoaderSpec spec(/*arity=*/3);
-    spec.AddInProcessSymbol(internal::GetAddI32CudaKernel(), "add");
+    spec.AddInProcessSymbol(stream_executor::gpu::internal::GetAddI32Kernel(), "add");
     TF_ASSERT_OK(executor->GetKernel(spec, &add));
   }
 
   {  // Load inc_and_cmp kernel.
     MultiKernelLoaderSpec spec(/*arity=*/3);
-    spec.AddInProcessSymbol(internal::GetIncAndCmpCudaKernel(), "inc_and_cmp");
+    spec.AddInProcessSymbol(stream_executor::gpu::internal::GetIncAndCmpKernel(), "inc_and_cmp");
```

Comment:
ditto

Reply:
@ezhulenev Sorry I've accidentally removed your review. Do you have time to check.